deploy/cr.yaml

```diff
@@ -5,6 +5,17 @@ metadata:
 spec:
   secretsName: cluster1-secrets
   sslSecretName: cluster1-ssl
+  backup:
+    image: percona/percona-xtrabackup:8.0.25-17
+    imagePullPolicy: Always
+    storages:
+      fs-pvc:
```

Comment:
Wasn't it mentioned that we will not have PVC backup in this release? Should we only have S3 in storages?